src/components/screens/EventRecords.js

```diff
@@ -200,15 +191,15 @@ export const EventRecords = ({ navigation, route }) => {
       showVolunteerEvents && refetchVolunteerEvents();
     }
     setRefreshing(false);
-  }, [isConnected, setRefreshing, showVolunteerEvents, refetch, refetchVolunteerEvents]);
+  }, [isConnected, setRefreshing, refetch, showVolunteerEvents, refetchVolunteerEvents]);
```

Comment:
Why did you need to change the sorting?

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/940/commits/f653378eefcca79cef1def80dfa038e995b1bccf